src/tcgwars/logic/impl/gen4/RisingRivals.groovy

```diff
@@ -1929,6 +1912,15 @@ public enum RisingRivals implements LogicCardInfo {
               draw 2
             }
           }
+          delayedA {
+            def warnAndPrevent = {
+              if (self.active && bg.currentTurn == self.owner.opposite) {
+                wcu "$self's Primal Law prevents playing Pokémon from your hand to evolve your Pokémon"
+                prevent()
+              }
+            }
+
+          }
```

Comment:
I think you were using this for reference and forgot to remove it?

Reply:
Indeed :(